nixos/modules/services/monitoring/osquery/flags.nix

```diff
@@ -0,0 +1,16 @@
+{ cfg, lib, pkgs }:
+
+with builtins;
```

Comment:
is there a good reason to have a separate file for this? I think it can be inlined to be honest, we have larger NixOS modules than this :P

Reply:
It has been inlined.